test/e2e/storage/drivers/csi.go

```diff
@@ -810,6 +811,7 @@ func InitGcePDCSIDriver() storageframework.TestDriver {
 				storageframework.CapVolumeLimits:        false,
 				storageframework.CapTopology:            true,
 				storageframework.CapControllerExpansion: true,
+				storageframework.CapOfflineExpansion:    true,
```

Comment:
Can we document what this capability means?

Reply:
The reason I am asking for offline expansion docs is - in this case offline capability being false is sort of unique to certain drivers that they can not be expanded if offline.

Reply:
Sure, I added a comment here where they're defined, let me know if there's a better place:
https://github.com/kubernetes/kubernetes/blob/f369b1234a5bd0fdf87e8697342890a8a6d22a30/test/e2e/storage/framework/testdriver.go#L170-L174